hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java

```diff
@@ -709,17 +766,24 @@ private static void usageSetTableState(PrintWriter writer) {
     writer.println("   An example making table name 'user' ENABLED:");
     writer.println("     $ HBCK2 setTableState users ENABLED");
     writer.println("   Returns whatever the previous table state was.");
+    writer.println("   If -i or --inputFiles is specified, pass one or more input file names.");
+    writer.println("   Each file contains <TABLENAME> <STATE>, one pair per line.);" +
+            "For example:");
+    writer.println("     $ HBCK2 -i " + SET_TABLE_STATE + " fileName1 fileName2");
   }
 
   private static void usageScheduleRecoveries(PrintWriter writer) {
-    writer.println(" " + SCHEDULE_RECOVERIES + " <SERVERNAME>...");
+    writer.println(" " + SCHEDULE_RECOVERIES + " <SERVERNAME|INPUTFILE_FOR_SERVERNAMES>...");
```

Comment:
```suggestion
    writer.println(" " + SCHEDULE_RECOVERIES + " <Either SERVERNAME OR INPUTFILE_FOR_SERVERNAMES>...");
```
As HBase expects | in some format like coprocessor so if we can be more descriptive that would be good.